modules/mod_translation/mod_translation.erl

```diff
@@ -494,9 +483,9 @@ language_enable(Code, IsEnabled, Context) ->
 
 
 %% @doc Set/reset the state of the 'rewrite URL' setting.
--spec set_language_url_rewrite(boolean(), #context{}) -> undefined.
+-spec set_language_url_rewrite(boolean(), #context{}) -> #context{}.
```

Comment:
The spec in line 486 uses a record, please define a type for the record and use that instead.